apps/condo/domains/acquiring/schema/RecurrentPaymentContext.js

```diff
@@ -104,6 +99,14 @@ const RecurrentPaymentContext = new GQLListSchema('RecurrentPaymentContext', {
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: async ({ resolvedData }) => {
+            //check at least one trigger are set
+            if (!resolvedData['autoPayReceipts'] && isNil(resolvedData['paymentDay'])) {
```

Comment:
Perhaps you should also check the presence of these fields in an already created item (when update operation)

Like  `const newItem = { ...existingItem, ...resolvedData }`
and use `newItem` to check `if (!newItem.someField) {...}`

Reply:
10x